### PR TITLE
Try out actionlint

### PR DESCRIPTION
**Pull request by ryanwholey**
Adds the [actionlint](https://github.com/rhysd/actionlint) script to lint our github action yaml. This should hopefully keep us from shooting ourselves in the foot one day. Once we are happy with this here I'll add an organization workflow for us over at `takescoop/.github`

### CODE REVIEW

File: .github/workflows/action-lint.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Lint GitHub Action
+on: [push]
+jobs:
+  action-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download actionlint
+        run: |
+          curl -fsSL https://github.com/rhysd/actionlint/releases/download/v1.5.3/actionlint_1.5.3_linux_386.tar.gz -o actionlint.tgz
```
Comment (bendrucker) [inline .github/workflows/action-lint.yml:10]:
```suggestion
          curl -fsSL https://github.com/rhysd/actionlint/releases/download/v1.5.3/actionlint_1.5.3_linux_mad64.tar.gz -o actionlint.tgz
```

Comment (bendrucker) [reply .github/workflows/action-lint.yml:10]: Would also be nice to move the version number out to an env for easier updating

**Review by bendrucker — CHANGES_REQUESTED** [review]
FYI:

https://github.com/rhysd/actionlint#ci-services

File: .github/workflows/action-lint.yml (hunk 1 above)
Comment (ryanwholey) [reply .github/workflows/action-lint.yml:10]: i saw the build script but figured it would be nice. env var would be nice

Comment (ryanwholey) [reply .github/workflows/action-lint.yml]: til 386 refers to the amd 32 bit version vs amd64 which is obviously for amd 64 bit systems

**Review by bendrucker — APPROVED** [review]

**Pull request merged**